completions/oc.completion.sh

```diff
@@ -0,0 +1,5 @@
+#! bash oh-my-bash.module
+if _omb_util_command_exists oc; then
+  eval -- "$(oc completion bash)"
+fi
+
```

Comment:
Thanks for updating the PR! and also this file.

Reply:
Whoops... done.

Reply:
Thanks!